internal/cmd/load-balancer/observability-credentials/list/list.go

```diff
@@ -92,6 +105,10 @@ func NewCmd(p *print.Printer) *cobra.Command {
 
 func configureFlags(cmd *cobra.Command) {
 	cmd.Flags().Int64(limitFlag, 0, "Maximum number of entries to list")
+	cmd.Flags().Bool(usedFlag, false, "List only used credentials")
```

Comment:
Maybe also rephrase to something like `List only credentials being used by a Load Balancer`